src/helpers/S3Helper.php

```diff
@@ -78,16 +78,16 @@ public static function uploadFileFromModel($model, UploadedFile $uploadedFile, s
 	/**
 	 * Метод для удаления файлов
 	 * @param int $storageId
+	 * @param string|null $bucket
 	 * @return int|null
 	 * @throws Throwable
 	 */
-	public static function deleteFile(int $storageId): ?int {
+	public static function deleteFile(int $storageId, ?string $bucket = null): ?int {
 		if (null === $storage = CloudStorage::findModel($storageId)) return null;
 		$storage->deleted = true;
-		if (false === $storage->save()) {
-			throw new ServerErrorHttpException('Could not record');
-		}
-		(new S3())->deleteObject($storage->key);
+		if (false === $storage->save()) return null;
```

Comment:
Размазанно получилось - null в данном случае связан с двумя несвязанными кейсами. Я предлагал тебе никак не реагировать на результат операции save(), т.к. считаю это излишним.
Призовем @pozitronik

Reply:
В принципе согласен, так как мало вероятно, что модель не сохранится при записи одного поля с булевым значением.

Reply:
Ну, например.
```suggestion
		if (false === $storage->save()) throw new Exception("Смерть жопа");
```
Неудача записи в БД как бы является реально непредусмотренной и катастрофической ситуацией, никак не относящейся к логике функции.
Но поскольку у нас вообще-то следит за этим фреймворк, то достаточно
```suggestion
		$storage->save();
```